pages/admin/index.tsx

```diff
@@ -11,80 +11,149 @@ import {
   Tab,
   Menu,
   MenuButton,
+  MenuList,
+  MenuItem,
   InputGroup,
   Input,
   InputLeftElement,
+  useControllableState,
+  Wrap,
 } from '@chakra-ui/react'; // Chakra UI
-import { AddIcon, ChevronDownIcon, SearchIcon } from '@chakra-ui/icons'; // Chakra UI Icons
+import { ChevronDownIcon, SearchIcon } from '@chakra-ui/icons'; // Chakra UI Icons
 import Layout from '@components/internal/Layout'; // Layout component
 import { Role } from '@lib/types'; // Role enum
 import { authorize } from '@tools/authorization'; // Page authorization
 import Table from '@components/internal/Table'; // Table component
 import Pagination from '@components/internal/Pagination'; // Pagination component
+import { RequestStatusBadge } from '@components/internal/RequestStatusBadge'; //Status badge component
+
+type StatusProps = {
+  value: any;
+};
```

Comment:
since status can only be a discrete set of values (variants in RequestStatusBadge) I think it would be a good idea to add those as a type instead of using `any`. Also it should be `readonly`